keyboards/mechmini/v1/keymaps/pitty/keymap.c

```diff
@@ -0,0 +1,56 @@
+/*
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "mechmini.h"
+#include "quantum.h"
+
+#define _QWERTY 0
+#define _NMBR 1
+#define _NAV 2
+#define _MOUSE 3
+#define _FUNCT 4
+
+#define _____ KC_TRNS
```

Comment:
QMK defines a built-in keycode for this automatically.

https://github.com/qmk/qmk_firmware/blob/76f0b3cc70d8d1173f9145abc4d2725fd8f47302/quantum/quantum_keycodes.h#L29-L31

It's much preferred to use the built-in keycode, rather than defining a different one here. Simply delete Line 23 and change all instances of `_____` to `_______`.